app-containers/skopeo/files/makefile-1.13.3.patch

```diff
@@ -0,0 +1,38 @@
+# Patch by Rahil Bhimjiani (IRC: rahilarious) (rahil3108@gmail.com)
+#
+# Rationale behind this patch:
+# 1. Policy.json and default.yaml which are already installed by app-containers/containers-common so to avoid conflict with skopeo prevent installing.
```

Comment:
Please include _why_ we're not installing them in this package too though.

Reply:
@thesamesam  Gentoo users are smart enough to know it's obvious, and these are devs whose avg IQ is >125. But anyway I added more -vvvvv to my english.  Felt like I was writing essay on r/explainlikeimfive :-)

Reply:
That's not a good answer, sorry.

It's not clear to me at all why skopeo installs the same configuration files as containers-common. I know containers-common installs them. You're not explaining why it's important that we use the version in there and not skopeo's upstream variant.

If we stopped using this patch, what would happen? Yes, a file collision, but beyond that? Why are we doing this whole config file dance?

You need to give clear rationale for patches so we know why they're there and it's just something we carry forever. That's how things become unmaintainable.

Reply:
@thesamesam In containers-common, we install these files from skopeo package itself [src](https://github.com/gentoo/gentoo/blob/master/app-containers/containers-common/containers-common-9999.ebuild#L57). Because these files are needed by podman, buildah & friends we install in containers-common and not in skopeo. 

Nothing will happen beyond file collision.

Reply:
Say that then! 

"We don't install these files as part of skopeo because podman, buildah, et. al need them as well, and there's no need for those to depend on skopeo just for configuration. Rely on app-containers/containers-common which has them split out instead. See https://bugs.gentoo.org/849863."

Easy.

Reply:
Thanks for the exact wordings. I have trouble in English to keep it short & direct. Apologies for my ineptness in expressing myself

Reply:
You don't need to apologise for the English - just understand that I wasn't asking to be pedantic, and that it wasn't obvious to everybody. Thank you!

Reply:
Done